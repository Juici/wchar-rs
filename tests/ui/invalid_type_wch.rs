use wchar::wch;

const INVALID_TYPE: &[f32] = wch!(f32, "oops");

fn main() {}
