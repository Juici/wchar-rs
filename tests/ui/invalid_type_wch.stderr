error: expected one of: `u16`, `u32`, `i16`, `i32`, string literal, character literal
 --> $DIR/invalid_type_wch.rs:3:35
  |
3 | const INVALID_TYPE: &[f32] = wch!(f32, "oops");
  |                                   ^^^
